team_comps.py

```diff
@@ -12,14 +16,27 @@ def get_team_data():
 
 
 def add_champ(champ_name):
-    user_team.append(champ_name)
-    print(user_team)
+    if valid_name(champ_name):
+        user_team.append(champ_name)
+        print(user_team)
+    else:
+        print("Champion %s is not valid, please enter a valid champion's name" % champ_name)
+        print(user_team)
+
 
+def delete_champ(champ_string):                            # -champion_name
+    champ_name_length = len(champ_string) - 1              # length = len(champion_name)
+    champ_name = champ_string[-champ_name_length:]
 
-def delete_champ(deletion_input):                            # -champion_name
-    champ_name_length = len(deletion_input) - 1              # length = len(champion_name)
-    user_team.remove(deletion_input[-champ_name_length:])    # remove("champion_name")
-    print(user_team)
+    if valid_name(champ_name):
```

Comment:
Is this if statement necessary? Wouldn't the nested if statement catch anything? For example, we enter in "" to delete, it would just return back it's not in the user list. No reason to check for that champion twice right?